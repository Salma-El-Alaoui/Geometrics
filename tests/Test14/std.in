LOAD test14-1.txt
LIST
CLEAR
LIST
LOAD test14-2.txt
LOAD test14-3.txt
LOAD test14-4.txt
EXIT
