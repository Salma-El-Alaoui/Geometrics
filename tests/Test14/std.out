OK
#Fichier charge
C c1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
R r1 1 1 1 1
OK
#Modele cleared
ERR
#Le fichier contient au moins deux elements ayant le meme nom.
ERR
#Le fichier contient au moins une commande ayant un type de parametre invalide.
ERR
#Le fichier contient au moins une commande ayant un nombre de parametres invalide.

